src/scripts/babel-gettext-extractor/index.js

```diff
@@ -19,6 +19,8 @@ const DEFAULT_HEADERS = {
 
 const DEFAULT_ADD_LOCATION = 'full'
 
+const NO_CONTEXT = 'no_context'
```

Comment:
**Issue:** If I remember correctly the "empty" context should always be empty, if not explicitly given.

```suggestion
const NO_CONTEXT = ''
```

Reply:
> **Issue:** If I remember correctly the "empty" context should always be empty, if not explicitly given.

It is needed only for reading fields without the context from replacements.json. It won't appear in the `message.pot` described it's purpose.